Yasgui/packages/yasqe/src/sparql.ts

```diff
@@ -118,24 +118,33 @@ export async function executeQueryModeQuery(yasqe: Yasqe, config?: YasqeAjaxConf
     yasqe.emit("query", req, populatedConfig);
     return await req.then(
       (result) => {
-        let hasMorePage = false;
-        if (!yasqe.isUpdateQuery() && !yasqe.isAskQuery() && yasqe.config.paginationOn) {
-          // If client hadn't set total Element we will execute count query.
-          if (!result.body.totalElements) {
-            executeCountQuery(yasqe, config);
-          } else {
-            yasqe.emit("totalElementChanged", parseInt(result.body.totalElements));
-          }
+        let hasMorePages = false;
+        let totalElements = result.body.totalElements ? parseInt(result.body.totalElements) : undefined;
```

Comment:
`result.body.totalElements` is this of a string type? If so, probably also move the parsing in the `if ` expression because otherwise it might return NaN which I'm not sure that it can be handled later.